README.md

```diff
@@ -1,25 +1,39 @@
 
-# OpenBoatUtils
-OpenBoatUtils makes boat physics much more customizable, and exposes those settings via Plugin Messages. This means that you can use commands in singleplayer to edit the settings, but in multiplayer the server will have control.
+## How It Works:
+OpenBoatUtils provides an interface for server administrators to adjust boat physics parameters. This API communicates with the server to update boat behavior in real-time, ensuring that all players experience the same physics adjustments. While OpenBoatUtils offers extensive customization options, its primary function is to serve as a tool for servers to implement these changes, rather than acting independently.
 
 ## Features:
-- Configurable step height
-- Configurable slipperiness
-- Configurable "Air Control"
-- Configurable Boat Jumping
-- Configurable gravity
-- Configurable acceleration values
-- more obscure nonsense
+- **Configurable step height:** Adjust how boats handle steps and inclines.
+- **Configurable slipperiness:** Modify the friction and sliding behavior of boats.
+- **Configurable "Air Control":** Control how boats behave when airborne.
+- **Configurable Boat Jumping:** Enable or adjust the ability for boats to jump.
+- **Configurable gravity:** Set the gravity affecting boats.
+- **Configurable acceleration values:** Customize how quickly boats accelerate.
+- **Additional customizable settings:** Explore more obscure and advanced settings for fine-tuning boat physics.
 
-There is more information about the features and how they work on the [wiki](https://github.com/o7Moon/OpenBoatUtils/wiki).
+For more detailed information about the features and how they work, visit the [wiki](https://github.com/o7Moon/OpenBoatUtils/wiki).
 
-For plugin developers, the wiki also has [documentation for all of the packets](https://github.com/o7Moon/OpenBoatUtils/wiki/Packets).
+For plugin developers, the wiki also includes [documentation for all of the packets](https://github.com/o7Moon/OpenBoatUtils/wiki/Packets).
+
+## Installation:
+To use OpenBoatUtils, you must have the [Fabric API](https://modrinth.com/mod/fabric-api) installed. Follow these steps to install obu:
+1. Download the latest version of OpenBoatUtils from the releases page.
+2. Ensure the Fabric API is installed on your server.
+3. Place the OpenBoatUtils JAR file into your server's mods folder.
+4. Start your server to load OpenBoatUtils.
+
+## Usage:
+Once installed, server administrators can use commands to adjust boat settings. In singleplayer, use the same commands to modify settings locally. Detailed command usage can be found on the [wiki](https://github.com/o7Moon/OpenBoatUtils/wiki).
```

Comment:
The intended purpose isn't to install obu on the server, and it's actually quite a bad idea to do so!

Maybe replace this section with:

> [!WARNING]
> OpenBoatUtils is not made to be installed on your fabric server! It is a client mod that your server's plugins/mods can use through a packet API. Installing the mod on your server **removes the movement anti-cheat** and **adds testing commands with no permissions**.